search-api/src/search_api/resources/v1/businesses/document_request.py

```diff
@@ -50,3 +54,33 @@ def get(business_identifier, request_id=None):
         return jsonify(documentAccessRequests=access_requests_list)
     except Exception as default_exception:  # noqa: B902
         return resource_utils.default_exception_response(default_exception)
+
+
+@bp.route('/<string:business_identifier>/documents/requests', methods=['POST', 'OPTIONS'])
```

Comment:
fyi I think you can simplify this to @bp.post(<route>) and then name the function below it something more meaningful if you want. Same with the gets above. Not actually sure if its better or not though haha - I did it for the search endpoints (I think its a new flask 2 thing?)